requirements-test-libraries.txt

```diff
@@ -105,6 +105,7 @@ shotgun_api3==3.5.1
 slixmpp==1.8.5
 spacy==3.8.2; python_version >= '3.10'
 srsly==2.4.8
+sv-ttk==2.6.0; python_version >= '3.8'
```

Comment:
There's no need for `; python_version >= '3.8'` since python 3.8 is the lowest version we support.